rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SortSpecifierRestContainer.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * ProActive Parallel Suite(TM): The Java(TM) library for
+ *    Parallel, Distributed, Multi-Core Computing for
+ *    Enterprise Grids & Clouds
+ *
+ * Copyright (C) 1997-2015 INRIA/University of
+ *                 Nice-Sophia Antipolis/ActiveEon
+ * Contact: proactive@ow2.org or contact@activeeon.com
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation; version 3 of
+ * the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this library; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307
+ * USA
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ *
+ * Initial developer(s):               The ProActive Team
+ *                         http://proactive.inria.fr/team_members.htm
+ */
+package org.ow2.proactive_grid_cloud_portal.common;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utility container to pass multiple parameters for tasks sorting.
+ *
+ * @author ActiveEon Team
+ */
+public class SortSpecifierRestContainer implements Serializable {
```

Comment:
@paraita This class is a duplicate of `SortSpecifierContainer`. Would be nice to avoid this duplication. You can probably make `rest-api` depend on `scheduler-apì`.